src/treelearner/cuda_kernel_launcher.cu

```diff
@@ -0,0 +1,164 @@
+/*!
+ * Copyright (c) 2020 IBM Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
+#ifdef USE_CUDA
```

Comment:
Not sure but do we need `#ifdef USE_CUDA` here if this file is included only for CUDA: 
```
if(USE_CUDA) 
    src/treelearner/*.cu 
endif(USE_CUDA)
```

Reply:
if we don't have that, then the code for building the R package will have to change. Since right now it relies on copying over all of `src/treelearner`

Reply:
for the CRAN package***

Reply:
ah never mind, I guess that fact that is uses `.cu` extension should be enough to keep the CRAN package from trying to build it